examples/custom-interactions/main.js

```diff
@@ -114,7 +114,8 @@ class CustomInteractionHandler {
   }
 }
 
-document.addEventListener('DOMContentLoaded', () => {
+document.addEventListener('DOMContentLoaded', async () => {
+  await window.customElements.whenDefined('vertex-viewer');
```

Comment:
It'd make more sense to me if this was part of `main`, since that where we're querying for the element that's waiting to be hydrated.